Assets/Scripts/InteractiveObject.cs

```diff
@@ -13,11 +13,13 @@ public class InteractiveObject : MonoBehaviour, IInteractive
     public string DisplayText => displayText;
     private AudioSource audioSource;
 
+    //On awake, find the audio source component of the object.
     protected virtual void Awake()
     {
         audioSource = GetComponent<AudioSource>(); 
     }
 
+    //Finds and plays the attached audio source, if there is no audio source catch and throw exception message.
     public virtual void InteractWith()
     {
         try
```

Comment:
Can't comment on lines below this, but please delete one of the extra spaces in lines 33/34, as this spacing is not repeated in other scripts.